js/webform_civicrm_payment.js

```diff
@@ -54,11 +54,18 @@ cj(function($) {
   });
   loadBillingBlock();
 
-  function tally() {
-    var total = 0;
+  function getTotalAmount() {
+    var totalFee = 0.0;
     $('.line-item:visible', '#wf-crm-billing-items').each(function() {
-      total += parseFloat($(this).data('amount'));
+      totalFee += parseFloat($(this).data('amount'));
     });
```

Comment:
nitpick: I think should probably be `totalAmount` (this could be a membership fee or an event fee - but it could also be a donation and certainly is not a credit card processing fee)

Reply:
@KarinG Resolved!